src/pymovements/utils/decorators.py

```diff
@@ -25,16 +25,16 @@
 from typing import Any
 
 
-def auto_str(cls):
+def auto_str(cls: Any) -> str:
```

Comment:
this is actually a decorator and should return the same class as the input class.

you can achieve this by:
```
ClassT = TypeVar("ClassT") 

def auto_str(cls: ClassT) -> ClassT:
```
see here:
https://stackoverflow.com/a/65613529

In the example they also use the following argument:
`bound=Callable[..., Any]`. I don't know if this is applicable in our case, as this decorator should only be used for classes not functions.

Reply:
done